src/logger/CreateLogEntry.ts

```diff
@@ -16,26 +16,22 @@ export function createLogEntry(logEntryInput: LogEntryInput): LogEntry {
     } = logEntryInput
 
     const logEntry: LogEntry = {
-        level: loglevel ?? LogLevel.info,
+        level: loglevel ?? 'INFO',
```

Comment:
I liked the old way better, was this changed for compatibility with other libraries?

Reply:
It was changed to allow the eraseable option in Typescript to be enabled. This makes it easier to strip types by removing them, and it does not work with enums, thus the change to string type.